Scripts/Python/ki/xKIConstants.py

```diff
@@ -130,14 +135,19 @@ class kAges:
                "Shaft" : "D'ni-Tiwah",
                "Spy Room" : "D'ni-Ae'gura",
                "spyroom" : "D'ni-Ae'gura",
+##             Fan Ages
                "Trebivdil" : "Tre'bivdil",
                "trebivdil" : "Tre'bivdil",
-               "vothol" : "Vothol Gallery"}
+               "vothol" : "Vothol Gallery",
+               "FehnirHouse" : "Fehnir's House",
```

Comment:
Vothol Gallery & Fehnir's House are both Cavern locations, no? They should be `D'ni-<something>`?

Reply:
Would this convention also extend to the various pubs?

Reply:
That's a good point dpogue. Let me test how that looks in Destiny.

Reply:
Don’t listen to my bikeshedding, but I don’t think this change is a proper continuation of the previous pattern. In the precedent, the “D'ni-” prefix is followed by a D’ni area name, not by an English description. So the pubs would all become “D'ni-J'taeri” (or whatever it was). The ambiguity would make it less useful though.
(In other words, the change makes things no less inconsistent than before. It does make them marginally more useful, so I guess it’s OK.)